lua/example_of_dynamic_config_request.lua

```diff
@@ -0,0 +1,15 @@
+local ffi = require("ffi")
+
+-- TODO Add support for other operations. For example:
+-- Functions to add/del/list the GK FIB entries.
+-- Functions to list the ARP table.
+-- Functions to list the ND table.
+-- Functions to process the GT policies.
+-- ......
+
+-- The client's Lua script should contain code to
+-- call the corresponding functions defined below.
```

Comment:
This file is to show what a request looks like. There's no problem in creating a library to support requests when we get something to add there.